packages/backend/src/app.ts

```diff
@@ -1,45 +1,48 @@
+/* eslint-disable no-shadow */
```

Comment:
disable no-shadow? Can we avoid this?

Reply:
do you prefer to name the errors in the callbacks differently?